code/src/components/AddToDo.js

```diff
@@ -1,32 +1,68 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable no-undef */
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
-import todos from 'redux/reducers/todos';
+import { todos } from 'redux/reducers/todos';
+import styled from 'styled-components';
 
-const AddToDo = () => {
+const NewToDo = styled.form`
+  line-height: 0.5rem;
+  padding-bottom: 0.8rem;
+`;
+
+const Input = styled.input`
+  background: #a83256;
+  border: none;
+  padding: 8px;
+  font-size: 18px;
+  font-family: 'Baloo 2', cursive;
+  border-bottom: 2px dashed;
+  :focus {
+    outline: none;
+  }
+`;
+
+const AddButton = styled.button`
+  font-family: 'Baloo 2', cursive;
+  font-size: 18px;
+  border: none;
+  background: transparent;
+  cursor: pointer;
+`;
+
+export const AddToDo = () => {
```

Comment:
nice that you like using styled-components! if you like different ways of styling you can read more about Sass or tailwind css :) Maybe you can also try to switch to rems, ems instead of px.